File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void)
{
    int n;
    //prompt for number 1-8 inclusively
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //draw line
    for (int i = 0; i < n; i++) //this is hight
    {
        for (int j = 0; j < n - 1 - i; j++) //this is dots in a row
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++) //this is # in a row
        {
            printf("#");
        }
        
        printf("\n");
    }
}

File: pset6/mario/mario.py
# Implement a program that prints out a half-pyramid of a specified height
# first prompt the user with get_int for the half-pyramid’s height, a positive integer between 1 and 8, inclusive

from cs50 import get_int

# prompt for number 1-8 inclusively
n = get_int("Height: ")
while True:
    if n < 1 or n > 8:
        n = get_int("Height: ")
    
    for i in range(n):
        print(" " * (n - 1 - i), end="")
        print("#" * (i + 1))
    break
